db/db_iterator_test.cc

```diff
@@ -3984,6 +3985,139 @@ TEST_P(DBIteratorTest, MemtableOpsScanFlushTriggerWithNext) {
     }
   }
 }
+
+TEST_P(DBIteratorTest, AverageMemtableOpsScanFlushTrigger) {
+  // Tests option memtable_avg_op_scan_flush_trigger with
+  // long tombstone sequences.
+  Random* r = Random::GetTLSInstance();
+
+  const int kAvgTrigger = 10;
+  Options options;
+  options.create_if_missing = true;
+  options.memtable_op_scan_flush_trigger = 500;
```

Comment:
Can we express memtable_op_scan_flush_trigger using kAvgTrigger to avoid relevant hard-coded value